packages/config/src/doc/IProfiInfoProps.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Should the file name be `IProfileInfoProps.ts`?

Reply:
renamed to `IProfInfoProps.ts` since all the other files are abbreviated like that 😋